modules/flight_interface/flight_interface.py

```diff
@@ -83,8 +88,14 @@ def run(self) -> "tuple[bool, odometry_and_time.OdometryAndTime | None]":
         )
         if not result:
             return False, None
-
         # Get Pylance to stop complaining
+
         assert odometry_local is not None
 
         return odometry_and_time.OdometryAndTime.create(odometry_local)
+
+    def apply_decision(self, decision: decision_command.DecisionCommand) -> None:
+        """
+        Apply the given decision command to the flight controller.
+        """
+        self.controller.upload_commands(decision)
```

Comment:
I think flight controller expects a list of dronekit.Command objects, not a DecisionCommand object, couple options here:
1. we could either change the function in common to accept a list of DecisionCommand objects and use enums to determine what dronekit.Command object to send (I like this option)
2. change this function to determine what dronekit.Command object to send based on enums
3. change the interface between flight interface and decision to pass dronekit.Command objects instead of DecisionCommand objects